src/components/CreatingOpenGameModal.tsx

```diff
@@ -51,7 +64,11 @@ export default class CreatingOpenGameModal extends React.PureComponent<Props> {
               ref={this.buyinInput}
             />
             <small className="form-text text-muted">
-              Enter an amount between 1 and 0.001
+            {
+              this.invalidBuyin
+              ? `Invalid buy in: ${this.invalidBuyin}. Please enter an amount between ${MIN_BUYIN} and ${MAX_BUYIN}`
+              : `Please enter an amount between ${MIN_BUYIN} and ${MAX_BUYIN}`
```

Comment:
Should we actually restrict input like so, or should we support smaller input than 0.001 ETH?

Reply:
I think you might also be able to do this at the html level using a number field with min/max. Might be worth a look.